lib/mini-lightbox.js

```diff
@@ -120,6 +159,40 @@
     };
 
     var cache = MiniLightbox._cache = {};
-
-    root.MiniLightbox = MiniLightbox;
+    
+    /**
```

Comment:
Trying to keep the code as clean as possible, I would recommend to just remove this part, as long there are no limitations (since we export the function). :boom: